ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-custom/query-custom.hbs

```diff
@@ -20,6 +20,9 @@
         </div>
         <div class="query-settings">
 
+        </div>
+        <div class="query-result-template">
```

Comment:
Why are we only allowing result detail templates for custom search templates?  I thought we would put this in the  settings component so that any search could utilize the result detail templates.

Reply:
This is the first time i've heard that.

Reply:
While the ticket was written before it was decided that `Basic` and `Text` were considered Search Forms, i'll refactor the PR.

Reply:
Are there any other refactors that need to be done in this PR?

Reply:
@andrewkfiedler Moved to result template selector to the search settings view.